strawberryfields/api/devicespec.py

```diff
@@ -97,17 +98,27 @@ def gate_parameters(self):
 
         return gate_parameters
 
+    def fill_template(self, program):
+        """Fill template with parameter values from a program"""
+        if program.type == "tdm" and program.timebins:
```

Comment:
Is it ever possible for `program.type == "tdm"` and not `progam.timebins`?

Reply:
No it isn't. I think I added this because before, `self.timebins` wasn't set in the `__init__` method, but it is now, so this can be removed here.